docs/en/contributor/Building-Alluxio-From-Source.md

```diff
@@ -17,6 +17,24 @@ This guide describes how to clone the Alluxio repository, compile the source cod
 - [Maven 3.3.9 or later](http://maven.apache.org/download.cgi)
 - [Git](https://git-scm.org/downloads)
 
+Alternatively, we have publishhed a docker image [alluxio/alluxio-maven](https://hub.docker.com/r/alluxio/alluxio-maven) with Java, Maven and Git installed to build Alluxio source code.
+For example, to build Alluxio with JDK8, checkout this image at tag `0.0.5-jdk8`:
+
+```console
+$ docker pull alluxio/alluxio-maven:0.0.5-jdk8
+```
+
+Create a container `alluxio-build` based on this image and get into this container to proceed:
+
+```console
+$ docker run -itd \
+  --network=host \
+  -v ${HOME}/.m2:/root/.m2 \
```

Comment:
do we want to keep a local copy of the maven cache if using docker to build? it will take longer in the future if we don't include, but people who are blindly following the instructions won't realize we're taking up disk space that won't be cleaned up after destroying the container

if you decide to include the git repo as a mount, then i think it makes sense to keep this. if it is an isolated container workflow, then i would prefer not to persist any files on the host machine